### DISCUSSION TITLE
Bezug der Impfquote

### DISCUSSION THREAD

**Discussion by thorstenleidl** [marcelrebmann/corona-widget-ios#7] [Q&A]
Hallo @marcelrebmann,
erst einmal ganz großes Lob für die nochmal überarbeitete Version deines Skripts. Ich hatte mir nun auch das Update lokal kopiert und bin sehr angetan von der schönen Übersicht. Eine Frage hätte ich nur zur Impfquote. Gibts einen Grund warum du dich auf die Erstimpfung beziehst und nicht auf die Gesamtquote? 
Ich hatte es direkt im Vergleich gesehen, da ich mir für Impfungen auch ein Skript geforkt hatte. Unten links im Screenshot sieht man Gesamt, Erst-, und Zweitimpfung inkl. Quote:
![DC5D93D9-E638-4E4F-B6E5-181DB971101D](https://user-images.githubusercontent.com/73391313/108161642-bd456800-70eb-11eb-9b80-0d44c3d8aa66.jpeg)

**Comment by marcelrebmann** [comment]
Hallo,
danke für dein Feedback zum neuen Design!

Da das Widget nur ein begrenztes Platzangebot bietet, müssen natürlich Prioritäten gesetzt werden.

Die Erstimpfung ist mMn am interessantesten, da sie einen guten Überblick gibt, wie viele Personen bereits "geschützt" sind und wie schnell die Impfung allgemein vorangeht.
Nach der ersten Impfung besteht ja bereits ein Schutz vor Infektion (und vor sehr schwerem Verlauf) und nach 3-4 Wochen nach Termin der Erstimpfung die Zweitimpfung der entsprechenden Person.
Somit ist die Aussage `X % der Bevölkerung haben bereits einen Impfschutz` möglich.

Bei der Gesamtquote (das Widget unten rechts im Screenshot) finde ich etwas problematisch, dass die Unterscheidung zwischen Erst- und Zweitimpfung untergeht - aus dem genannten Grund des Teilschutzes.
Bei der Berechnung kann dann nicht mehr auf die wirkliche Personenanzahl geschlossen werden.
`(Erstimpfungen + Zweitimpfungen) / Anzahl benötigte Impfungen (116,4Mio)`
Die Gesamtquote von 3,7% bedeutet dann eigentlich nicht, dass 3,7% der Bevölkerung in DE geimpft sind, sondern lediglich, dass 3,7% der insgesamt benötigten Impfdosen verimpft wurden.

**Reply by thorstenleidl** [reply]
Hallo @marcelrebmann,
danke für deine Rückmeldung. Mit der Argumentation verstehe ich deine Priorisierung. Durch die verschiedenen Widgets, welche ich nutze, kann ich mir dann den detaillierten Überblick holen.
